File: 122-Best-Time-to-Buy-and-Sell-Stock-II.java
class Solution {
    public int maxProfit(int[] prices) {
        int v = prices.length;
        int s = prices[0];
        int result = 0;

        for (int i=1 ; i < v;i++) {
            if (s < prices[i]) {
                result = result + prices[i] - s;

            }
            s = prices[i];

        }
        return result;


        
    }
}

File: 122-Best-Time-to-Buy-and-Sell-Stock-II.py
class Solution:
    def maxProfit(self, prices: List[int]) -> int:
        start = prices[0]
        len1 = len(prices)
        result = 0
        for i in range(0 , len1):
            if start < prices[i]: 
                result += prices[i] - start
            start = prices[i]
        return result
        